### DISCUSSION TITLE
Minor issue in exported pdf

### DISCUSSION THREAD

**Discussion by DarshanBanait** [anyproto/contributors#110] [Q&A] [answered]
I am not sure if the object type which is generally shown below the Title and description is supposed to be printed on the exported document as well. I exported a page in pdf and had that 'page' text in the content which seemed totally out of context in relation to the content of the document.
Is this how it was intended?
[MongoDB Syntax.pdf](https://github.com/anyproto/contributors/files/13477646/MongoDB.Syntax.pdf)

**Comment by fuksman** [comment, marked as answer]
Printing prints everything you see in your object. "Page" is an object type that is displayed because it's a featured relation. You can change the list of featured relations by clicking a triangle icon in the top right corner and then clicking stars next to the relations.
<img width="691" alt="image" src="https://github.com/anyproto/contributors/assets/2526238/690d971d-bc36-4d26-839a-cb8fce9f7ef7">

**Reply by DarshanBanait** [reply]
Yeah, its gone now.
Thanks.
